source/content/guides/sendgrid-wordpress-wp-mail-smtp.md

```diff
@@ -9,9 +9,9 @@ reviewed: "2020-11-02"
 permalink: docs/guides/:basename
 ---
 
-[SendGrid](https://sendgrid.com/) is a popular email service, but its [WordPress plugin](https://wordpress.org/plugins/sendgrid-email-delivery-simplified/) hasn't been updated in several years or tested with the latest 3 (as of this writing) major releases of WordPress.
+[SendGrid](https://sendgrid.com/) is a popular email service, but its [WordPress plugin](https://wordpress.org/plugins/sendgrid-email-delivery-simplified/) has been deprecated as of July 13, 2021.
 
-This guide will cover how to install and configure [WP Mail SMTP](https://wordpress.org/plugins/wp-mail-smtp/) to send emails from your WordPress site using Sendgrid
+This guide covers how to install and configure [WP Mail SMTP](https://wordpress.org/plugins/wp-mail-smtp/) to send emails from your WordPress site using Sendgrid.
```

Comment:
Maybe this could be explained a bit more. So the Sengrid WP plugin doesn't work, but how does the WP Mail SMTP  work with Sengrid?

Reply:
This isn't necessary maybe just a nice to have.